README.md

```diff
@@ -117,6 +117,25 @@ to the default Git context:
           push: true
 ```
 
+Building from the current repository automatically uses the [GitHub Token](https://docs.github.com/en/actions/security-guides/automatic-token-authentication),
+so it does not need to be passed. If you want to authenticate against another
+private repository for remote definitions, you can set the
+[`BUILDX_BAKE_GIT_AUTH_TOKEN` environment variable](https://docs.docker.com/build/building/variables/#buildx_bake_git_auth_token):
+
+> [!NOTE]
+> Supported since Buildx 0.14.0
+
+```yaml
+      -
+        name: Build and push
+        uses: docker/bake-action@v4
+        with:
+          source: "${{ github.server_url }}/${{ github.repository }}.git#${{ github.ref }}"
+          push: true
+        env:
+          BUILDX_BAKE_GIT_AUTH_TOKEN: ${{ secrets.MYTOKEN }}
+```
+
 ## Customizing
 
 ### inputs
```

Comment:
```suggestion
### Inputs
```

Reply:
I use lowercasing here to match the `action.yml`: https://github.com/docker/bake-action/blob/75d6ba252c60fbd6a267235d93ee614a9b2d2826/action.yml#L9

I'm not against changing casing but would need to do that on all GHA repos we have. Can we look at this in follow-up?